### ISSUE TITLE
[Popover] Accessibility

### ISSUE THREAD

**Issue by jeremie-lucca** [LuccaSA/lucca-front#2645]
Our component popover is not accessible with keyboard, which mean its content cannot be read or interacted with by some users.
